libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccount.java

```diff
@@ -452,7 +452,7 @@ public String getUserLevelStoragePath() {
 	 */
 	public String getCommunityLevelStoragePath() {
 		String leafDir = INTERNAL_COMMUNITY_PATH;
-		if (!TextUtils.isEmpty(communityId) && !communityId.equals(INTERNAL_COMMUNITY_ID)) {
+		if (!TextUtils.isEmpty(communityId) && !communityId.startsWith(INTERNAL_COMMUNITY_ID)) {
```

Comment:
@skyler-b What if the `communityId` starts with 15 chars of 0s and the last 3 are non-zero chars? I know it's highly unlikely, but our API has always expected and accepted only 18 char IDs. Maybe we should doc it better?